test/e2e/setup/setup.go

```diff
@@ -30,13 +31,13 @@ import (
 )
 
 const (
-	performCleanup = "PERFORM_CLEANUP"
-	deployDir      = "/workspace/config/manager"
-	roleDir        = "/workspace/config/rbac"
+	performCleanupEnv = "PERFORM_CLEANUP"
+	deployDirEnv      = "DEPLOY_DIR"
+	roleDirEnv        = "ROLE_DIR"
 )
 
 func Setup(t *testing.T) *e2eutil.Context {
-	ctx, err := e2eutil.NewContext(t, os.Getenv(performCleanup) == "True")
+	ctx, err := e2eutil.NewContext(t, envvar.ReadBool(performCleanupEnv))
```

Comment:
@rodrigovalin  we actually have a function that handles this already (including lower casing)